README.md

```diff
@@ -4,15 +4,19 @@ Example push notification server, written in Golang
 
 ## Project status
 
-This repo is very much a WIP, and some important features are not yet completed.
+![Status](https://camo.githubusercontent.com/47c9762c88d56b96ffa436e2af994dab07f6f61f2a0388cd08be7d42b1b8fef5/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f50726f6a6563745f5374617475732d446576656c6f7065725f507265766965772d79656c6c6f77)
 
-| Feature               | Status | Notes                                                        |
-| --------------------- | ------ | ------------------------------------------------------------ |
-| Installations Service | 🟢     | Some minor revisions will be needed, but looking pretty good |
-| Subscriptions Service | 🟢     |                                                              |
-| API Server            | 🟢     | Working as expected                                          |
-| XMTP Worker           | 🟡     | Needs more testing, but works                                |
-| Delivery Service      | 🟡     | Basic implementation in place. Untested                      |
+This project is in Developer Preview. Software in this phase does is not formally supported, will change without warning, and may not be backwards compatible. We do NOT recommend using Developer Preview software in production applications.
```

Comment:
What do you think of this phrasing @neekolas? @yash-luna suggested replacing the explicit warnings with softer language indicating that it is OK to start building, just not in production apps. Do you think this softer language accurately reflects where the notification server is right now - or are you more comfortable with the explicit statements? =) 

```suggestion
This project is in Developer Preview status and ready to serve as a reference for you to start building. 

However, we do NOT recommend using Developer Preview software in production apps. Software in this status may change based on feedback.
```

Reply:
FWIW, I got the phrasing directly from @yash-luna’s doc. Should we update the language there so we have consistency across all projects?